docs/devGuide/devGuide.md

```diff
@@ -136,20 +136,41 @@ $ npm run test
 For Windows users:
 
 ```
-$ npm run testwin
+> npm run testwin
 ```
 
 #### Updating tests
 
 When adding new features, updating existing features or fixing bugs, you should update the expected site to reflect the changes.
 
+You may use the following script to update the expected sites:
+
+For Unix:
+
+```
+$ npm run updatetest
+```
+
+For Windows users:
+
+```
+> npm run updatetestwin
+```
+<box type="warning">
+  You should always check that the output generated is correct before committing your changes to the expected site.
+</box>
+
 ##### Changes to existing features
 
-Simply update the expected HTML files in `test/test_site/expected/` to reflect the changes.
+Simply update the expected HTML files for each test site to reflect the changes.
 
 ##### New features
 
-Add new site content into the `test/test_site/` folder to demonstrate the new feature. Ensure that the new content is included in the test site so that your feature will be tested when `markbind build` is run on the test site. Remember to update the expected HTML files in `test/test_site/expected/`.
+Add new site content into an existing test site or create a new test site to demonstrate the new feature. Ensure that the new content is included in the test site so that your feature will be tested when `markbind build` is run on the test site. Remember to update the expected HTML files for the test site.
```

Comment:
I feel like there's some repetition here. The special case of adding new site content / creating a new test site only applies when adding a new feature.

So maybe we can remove the `Changes to existing features` section, and combine `Simply / Remember to update the expected files...` with L144-146.

Say, something like:
```
#### Updating tests

Whether you are adding a new feature, updating existing features or fixing bugs, make sure to update the test sites to reflect the changes. You may use the following script to do this automatically:

[...]

#### When adding new features

Add new site content into an existing test site or create a new test site to demonstrate the new feature. This is to ensure that your feature can be tested by building that test site.

<box type"info">
  When creating a new test site...
```

Reply:
Also, dropping the `HTML` here because sometimes the non-HTML files get modified too